coriolis/providers/replicator.py

```diff
@@ -519,10 +535,10 @@ def _copy_file(self, ssh, localPath, remotePath):
         except IOError as ex:
```

Comment:
We have a `test_ssh_path` utility in the `coriolis.utils` module, could you please use that?